src/hal/utils/scope_vert.c

```diff
@@ -647,7 +647,7 @@ static void init_vert_info_window(void)
     /* turn off the channel if button is clicked */
     g_signal_connect(button, "clicked",
 	G_CALLBACK(channel_off_button), NULL);
-    gtk_widget_show(button);
+    // gtk_widget_show(button);
```

Comment:
Probably better to just delete this line, and the ones above it where the "Chan Off" button is created.

Reply:
It's probably better to have just one way to turn off a channel (by clicking on the enabled channel button).  If someone complains persuasively about the change we can always resurrect the old "Chan Off" button code out of git.

Reply:
Okay, I removed the button code.